Shared/Extensions/UIImageExtensions.swift

```diff
@@ -61,9 +61,10 @@ extension UIImage {
         UIGraphicsEndImageContext()
         return scaledImage!
     }
-
-    public static func templateImageNamed(_ name: String) -> UIImage? {
-        return UIImage(named: name)?.withRenderingMode(.alwaysTemplate)
+    
+    /// Return a UIImage which will always render as a template
+    public var template: UIImage {
+        return withRenderingMode(.alwaysTemplate)
```

Comment:
You have not used this helper in all images you added, there's still few with `withRenderingMode(.alwaysTemplate)`

Reply:
See my previous comment with Terry's review 🙂

> I have a changeset ready that updates all the UIImage's in the project to use #imageLiteral if available, but it involves a change made in this PR so a separate PR will be added once this PR is merged :)

Reply:
My bad, code review race condition :(